### PR TITLE
Model test

### PR DESCRIPTION
**Pull request by rongncao**
Prepare_model_data.py: read raw data from local directory and prepare train/valid/test dataset
Models.py: all neural network model implemented in pytorch
Main.py: train the model and print the accuracy and f1score, it requires the dataset from Prepare_model_data.py

### CODE REVIEW

File: Sleep_stage_classifier/shhs/polysomnography/polysomnography_reader.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import mne
+from shhs.parser import xml_nsrr as xn
```
Comment (macnb9) [inline Sleep_stage_classifier/shhs/polysomnography/polysomnography_reader.py:2]: you should be able to do this without copying the same files into your own submodule, referencing the original files instead. We def don't want duplicate files across this

**Review by macnb9 — CHANGES_REQUESTED** [review]
You should delete the copies that you made of the `shhs` directory and its files and reference its module directly - see my PR #15 if you want to see an example
